src/Apps/Collect/Components/Base/CollectFilterContainer.tsx

```diff
@@ -0,0 +1,94 @@
+import { CollectFilterContainer_viewer } from "__generated__/CollectFilterContainer_viewer.graphql"
+import { ContextConsumer } from "Artsy"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Responsive } from "Utils/Responsive"
+import { CollectRefetchContainer } from "./CollectRefetch"
+
+import { FilterContainer } from "../Filters"
+
+export interface CollectFilterContainerProps {
+  viewer: CollectFilterContainer_viewer
+}
+export class CollectFilterContainer extends Component<
+  CollectFilterContainerProps
+> {
+  render() {
+    const { filter_artworks } = this.props.viewer
+    const { aggregations } = filter_artworks
+    const mediumAggregation = aggregations.find(agg => agg.slice === "MEDIUM")
+
+    return (
+      <ContextConsumer>
+        {({ user, mediator }) => {
+          return (
+            <Responsive>
+              {({ xs }) => {
+                return (
+                  <FilterContainer
+                    isMobile={xs}
+                    user={user}
+                    mediator={mediator}
+                    mediums={mediumAggregation.counts as any}
+                  >
+                    {({ filters }) => (
+                      <CollectRefetchContainer
```

Comment:
The convention we've been following in other relay app code is to typically export the full name + relay wrapper type and then rename at the import so as to not encode relay behavior into runtime code.